guest-verona-rt/test/bench/boot/main.cc

```diff
@@ -7,22 +7,25 @@
  * Extract using `nm {ElfFile} | grep __unloaded_start`.
  * Subtract 0x600000 which is the ELF start address.
  */
-constexpr size_t BASE_LOADED_IMAGE = 0x494000;
-constexpr size_t TARGET_LOADED_IMAGE = BASE_LOADED_IMAGE;
+constexpr size_t BASE_LOADED_IMAGE = 0x92f000 - 0x600000;
+constexpr size_t TARGET_LOADED_IMAGE = 1024 * 1024 * 4;
+static_assert(
+  BASE_LOADED_IMAGE < TARGET_LOADED_IMAGE,
```

Comment:
Would this not be better at "<="? Which is what to set for `TARGET_LOADED_IMAGE` to identify the minimum.

Reply:
Yes, sorry, that got lost in my using that example for debugging, and I didn't spot the change.